### ISSUE TITLE
WCAG: Mandatory fields are not read out as required

### ISSUE THREAD

**Issue by Amit-Ag** [nextcloud/android#8751]
### Steps to reproduce
1.  Open https://stackblitz.com/angular/rxpkdamqnqp?file=src/app/input-errors-example.ts
2.  Open Preview
3.  Enable Talkback
4.  Focus on the Email input field

### Expected behaviour
- Talkback should announce the field as `required` as it is a mandatory field

### Actual behaviour
- Talkback does not announce the field as `required`

### Environment data
Android version: 9

Device model:  ONEPLUS A3003

Stock or customized system: OxygenOS Version 9.0.6

Test report : 

1. https://a11ysupport.io/tests/tech__aria__aria-required-on-html-radio-buttons/aria__aria-required_attribute/convey_value_true/talkback/and_chr
2. https://a11ysupport.io/tests/tech__aria__aria-required/aria__aria-required_attribute/convey_value_true/talkback/and_chr

**Comment by AndyScherzinger** [comment]
@Amit-Ag yopu are sure you are in the right repository/project? The link you are refering to has nothing to do with Nextcloud!

**Comment by Amit-Ag** [comment]
@AndyScherzinger  Apologies, mistakenly reported the bug here.
